docs/code-quality/ca1303.md

```diff
@@ -35,14 +35,14 @@ A method passes a string literal as a parameter to a .NET constructor or method
 
 This warning is raised when a literal string is passed as a value to a parameter or property and one or more of the following cases is true:
 
-- The <xref:System.ComponentModel.LocalizableAttribute> attribute of the parameter or property is set to true.
-
-- The parameter or property name contains "Text", "Message", or "Caption".
+- The <xref:System.ComponentModel.LocalizableAttribute> attribute of the parameter or property is set to true. 
 
 - The name of the string parameter that is passed to a Console.Write or Console.WriteLine method is either "value" or "format".
 
 By default, this rule analyzes the entire codebase, but this is [configurable](#configurability).
 
+Starting with version 3 of the rule, the naming heuristic is no longer used by default but this is [configurable](#configurability).
```

Comment:
There is a breaking change compared to previous version so I am mentioning the version here. If that's not how you want to handle it please feel free to edit.

Reply:
Probably both the hyperlinks for configuration options should now go to the respective sub-sections instead of `#configurability`?

Reply:
I am not sure to get which change you want here?!

Reply:
Should this be: `[configurable](#use-naming-heuristic)`?